packages/lesmis/src/components/MapOverlaysPanel/MapOverlaysPanel.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { MapOverlayGroup } from './MapOverlayGroup';
+import { MapOverlaysLoader } from './MapOverlaysLoader';
+import { MapOverlaysPanelContainer as Container } from './MapOverlaysPanelContainer';
+
+const Body = styled.div`
+  padding: 0.2rem 0 2rem;
+`;
+
+export const MapOverlaysPanel = ({ overlays, isLoading, overlayId, setOverylayId }) => {
+  const [selectedPath, setSelectedPath] = useState(null);
```

Comment:
Part of the design is to highlight the path to the selected overlay. This is the state that stores that path so that the nested components can be highlighted accordingly. example path eg. [0, 0, 1]